modules/core/src/v2/coins/algo.ts

```diff
@@ -1,11 +1,15 @@
 import BaseCoin = require('../baseCoin');
```

Comment:
Can you configure prettier on this file?

You do that by adding this at the top of the file:
```
/*
 * @prettier 
 */
```

Then your code gets auto formatted upon commit.

Reply:
sure